packages/react-magma-docs/src/pages/design/loading-indicators.mdx

```diff
@@ -7,16 +7,15 @@ order: 1
 import { Link } from 'gatsby';
 import './design-guidelines-styles.css';
 import DocsHeading from '../../components/docs-heading';
-import { Paragraph } from 'react-magma-dom';
+import LeadParagraph from '../../components/lead-paragraph';
```

Comment:
I know we've been a bit inconsistent on this (the DocsHeading above is the same), but I'd like to move towards standardizing on named exports. Those are less likely to be renamed and that should help keep things consistent and lighten cognitive load when trying to follow the code.

Reply:
Sounds good -- yes, I was wondering about that myself.  I think I created the DocsHeading a long time ago, and I made this one match.   There are a few others in the docs as well.  I'd like to do this as a separate PR from this one, if you're okay with that.